external-crates/move/crates/move-compiler/src/parser/syntax.rs

```diff
@@ -274,46 +324,121 @@ fn parse_comma_list_after_start<F, R>(
     start_loc: usize,
     start_token: Tok,
     end_token: Tok,
+    item_first_set: &TokenSet,
```

Comment:
This name feels meaningless to me. How am I to interpret it?

Reply:
https://www.cs.uaf.edu/~cs331/notes/FirstFollow.pdf A `First` set is the set of symbols that can start a production. This is `FIRST(<Item>)`. Happy for name suggestions, though!